src/components/Wheel/index.js

```diff
@@ -0,0 +1,129 @@
+import React, { useRef, useEffect, useState } from 'react';
+
+import ModalWindow from '../ModalWindow';
+import fortune_data from '../../data/fortune.json';
+import Button from '@material-ui/core/Button';
+
+import './index.css';
+
+function Wheel() {
+  const [rotateDegree, setRotateD] = useState(0.0);
+  const [modalProps, setModalProps] = useState({
+    isActive: false,
+    questionText: '',
+  });
+
+  function timer(ms) {
+    return new Promise((resolve) => setTimeout(resolve, ms));
+  }
+
+  function question(deg) {
+    setModalProps({ isActive: false, questionText: '' });
+    setRotateD(deg);
+    const questionNumber = Math.floor(
+      fortune_data.length - (deg % 360) / (360 / fortune_data.length),
+    );
+    console.log(fortune_data[questionNumber].question);
+    setModalProps({
+      isActive: true,
+      questionText: fortune_data[questionNumber].question,
+    });
+  }
+
+  async function spin() {
+    const rounds = Math.ceil(Math.random() * 2000 + 3000);
+
+    for (let i = rotateDegree; i <= rotateDegree + rounds; ) {
+      if (i === rotateDegree + rounds) {
+        question((rotateDegree + rounds) % 360);
+      } else {
+        setRotateD(i % 360);
+      }
+      i++;
```

Comment:
why do you use `i++` here instead of loop statement? (fe, `for (let index = 0; index < 10; index += 1)`)